ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-commands.sh

```diff
@@ -25,6 +26,51 @@ function getlogs() {
 # Gather logs regardless of what happens after this
 trap getlogs EXIT
 
+echo '#### Gathering Sos reports from all Nodes'
+
+timeout -s 9 30m ssh "${SSHOPTS[@]}" "root@${IP}" bash - << "EOFTOP"
+    if [ $(virsh list --name |  tr -s '\n'  | wc -l) > 0 ]
+    then
+      export SOS_BASEDIR="/tmp/artifacts/sos"
+      export HOSTFILE="${SOS_BASEDIR}/virtual_hosts.json"
+      mkdir -p "${SOS_BASEDIR}"
+    
+      export VIRT_NET=$(virsh net-list --name  | grep -v default | tr -d "\n")
+      export TMP_HOSTFILE="/tmp/tmp_hostfile.json"
+
+      echo "[]" > ${TMP_HOSTFILE}
+      virsh net-dhcp-leases --network "${VIRT_NET}" | grep 'master\|worker' | awk '{ printf("%s/%s\n", $6, $5) }' | while read -r line
+      do
+          _HOST=$( echo "$line" | cut -d"/" -f1 )
+          _IP=$( echo "$line" | cut -d"/" -f2 )
+    
+          jq """. += [{ \"host\" : \"${_HOST}\", \"address\" : \"${_IP}\" }]""" < ${TMP_HOSTFILE} > ${HOSTFILE}
+          cat "${HOSTFILE}" > "${TMP_HOSTFILE}"
+      done
+    
+      jq -c ".[]" ${HOSTFILE} | while read ITEM 
+      do
+        _HOST=$(echo ${ITEM} | jq -r .host )
+        _IP=$(echo ${ITEM} | jq -r .address )
+
+        export REPORT_PATH="${SOS_BASEDIR}/${_HOST}"
+        mkdir -p ${REPORT_PATH}; chmod o+w ${REPORT_PATH} 
+
+        timeout -s 9 30m ssh -o StrictHostKeyChecking=off -l core $_IP REPORT_PATH="${SOS_BASEDIR}/${_HOST}" bash - << "EOF"
+          mkdir -p ${REPORT_PATH} 
+    
+          yes | toolbox "sos report --batch --tmp-dir ${REPORT_PATH} -k crio.all=on -k crio.logs=on -k openshift.host=on -k openshift.podlogs=on \
+              -o  openshift,openshift_ovn,crio,containers_common,host,containerd,logs"
+
+            sudo chmod -R +r "${REPORT_PATH}"
```

Comment:
```suggestion
           sudo chmod -R +r "${REPORT_PATH}"
```
nit, looks like it's not aligned